Enhance SABnzbd documentation with use cases, examples and troubleshooting tips

jpbede opened the pull request:
## Proposed change
Enhance the SABnzbd integration documentation with use cases, examples and some troubleshooting tips

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

jpbede committed:
Enhance SABnzbd documentation with use cases, examples and troubleshooting tips

jpbede committed:
Fix example

jpbede committed:
Fix list

c0ffeeca7 commented on source/_integrations/sabnzbd.markdown:
```diff
@@ -20,54 +20,231 @@ ha_integration_type: integration
+- **Disk space monitoring**: Set up alerts when your download drive is running low on space
+- **Smart scheduling**: Automatically start downloads during off-peak hours when internet is faster or cheaper
+- **Home theater integration**: Pause downloads when movie night starts to ensure smooth streaming
+- **Security monitoring**: Get alerted if SABnzbd goes offline or becomes unreachable
```
```suggestion
- **Download completion notifications**: Get notified on your phone or smart display when downloads finish.
- **Bandwidth management**: Automatically pause downloads during peak internet usage hours or when streaming services are active.
- **Disk space monitoring**: Set up alerts when your download drive is running low on space.
- **Smart scheduling**: Automatically start downloads during off-peak hours when internet is faster or cheaper.
- **Home theater integration**: Pause downloads when movie night starts to ensure smooth streaming.
- **Security monitoring**: Get alerted if SABnzbd goes offline or becomes unreachable.
```

c0ffeeca7 commented on source/_integrations/sabnzbd.markdown:
```diff
@@ -20,54 +20,231 @@ ha_integration_type: integration
-- Copy your API key under "Security".
+1. Navigate to your SABnzbd web interface
+2. Click **Config** {% icon "mdi:settings" %}, then click **General**
+3. Copy your API key under **Security**
```
```suggestion
1. Navigate to your SABnzbd web interface.
2. Select *Config** {% icon "mdi:settings" %}, then **General**.
3. Copy your API key under **Security**.
```

c0ffeeca7 commented on source/_integrations/sabnzbd.markdown:
```diff
@@ -20,54 +20,231 @@ ha_integration_type: integration
+
+### Binary sensors
+
+- **Warnings**: Indicates if SABnzbd has any warnings (e.g., disk space low, download errors)
```
```suggestion
- **Warnings**: Indicates if SABnzbd has any warnings (for example, disk space low, download errors)
```

c0ffeeca7 commented on source/_integrations/sabnzbd.markdown:
```diff
@@ -20,54 +20,231 @@ ha_integration_type: integration
+
+### Numbers
+
+- **Speed limit**: Set the download speed limit (as a percentage of your configured maximum speed)
```
```suggestion
- **Speed limit**: Set the download speed limit (as a percentage of your configured maximum speed).
```

c0ffeeca7 commented on source/_integrations/sabnzbd.markdown:
```diff
@@ -20,54 +20,231 @@ ha_integration_type: integration
+3. **Test API key**: Verify your API key is correct by comparing it in the SABnzbd web interface
+4. **Network connectivity**: If SABnzbd is on another device, ensure Home Assistant can reach it over the network
+5. **Firewall settings**: Check that your firewall allows connections to SABnzbd's port
+6. **Enable debug logging**: Temporarily enable [debug logging](/docs/configuration/troubleshooting/#debug-logs-and-diagnostics) for the SABnzbd integration to get more detailed error messages.
```
```suggestion
1. **Verify SABnzbd is running**: Check that SABnzbd is running and accessible via its web interface.
2. **Check the URL**: Ensure you're using the correct URL format including the port (typically `http://localhost:8080`).
3. **Test API key**: Verify your API key is correct by comparing it in the SABnzbd web interface.
4. **Network connectivity**: If SABnzbd is on another device, ensure Home Assistant can reach it over the network
5. **Firewall settings**: Check that your firewall allows connections to SABnzbd's port.
6. **Enable debug logging**: Temporarily enable [debug logging](/docs/configuration/troubleshooting/#debug-logs-and-diagnostics) for the SABnzbd integration to get more detailed error messages.
```

c0ffeeca7 commented on source/_integrations/sabnzbd.markdown:
```diff
@@ -20,54 +20,231 @@ ha_integration_type: integration
+
+1. **Use internal URL**: Use `http://a02368d7-sabnzbd:8080` instead of `localhost`
+2. **Check add-on logs**: Review the SABnzbd add-on logs for any error messages
+3. **Add-on configuration**: Ensure the add-on is properly configured and started
```
```suggestion
1. **Use internal URL**: Use `http://a02368d7-sabnzbd:8080` instead of `localhost`.
2. **Check add-on logs**: Review the SABnzbd add-on logs for any error messages.
3. **Add-on configuration**: Ensure the add-on is properly configured and started.
```

c0ffeeca7 committed:
tiny tweaks

End sentences with a full stop, replace e.g. by for example, replace click by select.

c0ffeeca7 commented on source/_integrations/sabnzbd.markdown:
```diff
@@ -20,54 +20,231 @@ ha_integration_type: integration
+
+Here are some practical ways you can use the SABnzbd integration:
+
+- **Download completion notifications**: Get notified on your phone or smart display when downloads finish.
```
we use bold to mark UI strings. I don't think it should be used for other kinds of highlighting.

jpbede replied:
Fixed 🙂

jpbede committed:
remove bold

c0ffeeca7 commented on source/_integrations/sabnzbd.markdown:
```diff
@@ -20,54 +20,231 @@ ha_integration_type: integration
+
+### SABnzbd not found or unreachable
+
+1. **Verify SABnzbd is running**: Check that SABnzbd is running and accessible via its web interface.
```
same here. Or are these UI strings that come up when the issue occurs?

jpbede replied:
You're absolutely right. Overlooked this 🙈

jpbede committed:
remove bold

c0ffeeca7 reviewed (approved):
Thank you, @jpbede :+1:
